scripts/_Team_Defines.bash

```diff
@@ -71,6 +62,38 @@ alias setup-repository-ci=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/setup-ci-config.bash
 
 alias setup-ros-workspace=$RosTeamWS_FRAMEWORK_SCRIPTS_PATH/setup-ros-workspace.bash
 
+setup-ros-workspace () {
```

Comment:
Those functions should be moved to "_RosTeamWS_Docker_Devines" file because "_Team_Defines" should be developers' team specific setup and configuration

Reply:
Done